### PR TITLE
Fix poison HitsplatID. Add other Hitsplats.

### PR DESCRIPTION
**Pull request by SuperNerdEric**

### CODE REVIEW

File: runelite-api/src/main/java/net/runelite/api/HitsplatID.java [hunk 1]
```diff
@@ -47,4 +49,12 @@ public final class HitsplatID
 	public static final int DAMAGE_MAX_ME_ORANGE = 45;
 	public static final int DAMAGE_MAX_ME_YELLOW = 46;
 	public static final int DAMAGE_MAX_ME_WHITE = 47;
+	public static final int DAMAGE_ME_POISE = 53;
+	public static final int DAMAGE_OTHER_POISE = 54;
+	public static final int DAMAGE_MAX_ME_POISE = 55;
```
Comment (SuperNerdEric) [inline runelite-api/src/main/java/net/runelite/api/HitsplatID.java:54]: image

File: runelite-api/src/main/java/net/runelite/api/HitsplatID.java [hunk 2]
```diff
@@ -30,10 +30,12 @@ public final class HitsplatID
 	public static final int BLOCK_OTHER = 13;
 	public static final int DAMAGE_ME = 16;
 	public static final int DAMAGE_OTHER = 17;
-	public static final int POISON = 2;
+	public static final int POISON = 65;
 	public static final int DISEASE = 4;
 	public static final int VENOM = 5;
 	public static final int HEAL = 6;
+	public static final int CYAN_UP = 11;
+	public static final int CYAN_DOWN = 15;
```
Comment (SuperNerdEric) [inline runelite-api/src/main/java/net/runelite/api/HitsplatID.java:38]: image

File: runelite-api/src/main/java/net/runelite/api/HitsplatID.java [hunk 3]
```diff
@@ -30,10 +30,12 @@ public final class HitsplatID
 	public static final int BLOCK_OTHER = 13;
 	public static final int DAMAGE_ME = 16;
 	public static final int DAMAGE_OTHER = 17;
-	public static final int POISON = 2;
+	public static final int POISON = 65;
```
Comment (SuperNerdEric) [inline runelite-api/src/main/java/net/runelite/api/HitsplatID.java:33]: Everywhere I checked poison was actually 65, not sure if they moved them around at some point.

**Comment by SuperNerdEric** [comment]
> Is there a reason you have them named "poise" instead of "poison"?

Those are what the Wiki calls poise for the Warden's. Not sure if you want to name them by their color or shape or something like some of the others. I'm okay with renaming the constants to whatever.

**Comment by Adam-** [comment]
Ah, "poise" is a distinct hitsplat type from "poison".

**Pull request merged**